src/v/redpanda/application.cc

```diff
@@ -1496,6 +1496,27 @@ void application::start_runtime_services(
     _group_manager.invoke_on_all(&kafka::group_manager::start).get();
     _co_group_manager.invoke_on_all(&kafka::group_manager::start).get();
 
+    // Initialize the Raft RPC endpoint before the rest of the runtime RPC
+    // services so the cluster seeds can elect a leader and write a cluster
+    // UUID before proceeding with the rest of bootstrap.
+    const bool start_raft_rpc_early = !stored_cluster_uuid.has_value();
```

Comment:
I think this should also be tied to cluster discovery via some `is_founding_cluster()` call or somesuch -- non-seeds probably don't need to do this.

Reply:
Changing to
```
    const bool start_raft_rpc_early = !stored_cluster_uuid.has_value()
                                      && cd.is_cluster_founder().get();
```